From cc95fb4b1635bd1018d74b668430cda67731148f Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@konsulko.com>
Date: Sat, 30 Apr 2022 18:37:46 +0000
Subject: [PATCH] rgxinit.c: cast boolean value to IMG_BOOL

Fixes this error with GCC 11:

| .../services/server/devices/rogue/rgxinit.c:1345:36: error: implicit conversion from 'enum <anonymous>' to 'IMG_BOOL' {aka 'enum tag_img_bool'} [-Werror=enum-conversion]
|  1345 |                         bEnableAPM = false;
|       |                                    ^

Upstream-Status: Pending

Signed-off-by: Denys Dmytriyenko <denys@konsulko.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 services/server/devices/rogue/rgxinit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/server/devices/rogue/rgxinit.c b/services/server/devices/rogue/rgxinit.c
index ca7a1b9..fc94b73 100644
--- a/services/server/devices/rogue/rgxinit.c
+++ b/services/server/devices/rogue/rgxinit.c
@@ -1342,7 +1342,7 @@ PVRSRV_ERROR RGXInitDevPart2(PVRSRV_DEVICE_NODE	*psDeviceNode,
 		if (bEnableAPM && (!PVRSRV_VZ_MODE_IS(NATIVE)))
 		{
 			PVR_DPF((PVR_DBG_WARNING, "%s: Active Power Management disabled in virtualization mode", __func__));
-			bEnableAPM = false;
+			bEnableAPM = (IMG_BOOL)false;
 		}
 
 #if defined(RGX_NUM_OS_SUPPORTED) && (RGX_NUM_OS_SUPPORTED > 1) && defined(SUPPORT_AUTOVZ)
-- 
2.25.1

